docs/public/_redirects

```diff
@@ -525,11 +525,12 @@ https://v4.material-ui.com/* https://v4.mui.com/:splat 301!
 ## MUI X
 ## Unlike the store that expect to be hosted under a subfolder,
 ## MUI X is configured to be hosted at the root.
-/static/x/* https://material-ui-x.netlify.app/static/x/:splat 200
-/x/_next/* https://material-ui-x.netlify.app/_next/:splat 200
-/x/* https://material-ui-x.netlify.app/x/:splat 200
-/r/x-* https://material-ui-x.netlify.app/r/x-:splat 200
-/:lang/x/* https://material-ui-x.netlify.app/:lang/x/:splat 200
+/static/x/* https://docs-v6--material-ui-x.netlify.app/static/x/:splat 200
```

Comment:
This link seems wrong. Currently mui.com points to v7 of the DataGrid, not v6, while https://docs-v6--material-ui-x.netlify.app/x/introduction/ points to the v6 of the DataGrid.

Reply:
I think this is correct. These redirects are for DataGrid users who want to see v6 docs.

Ex, access to `https://v6.mui.com/x/react-data-grid/*`, request assets to `https://v6.mui.com/static/x/* should redirect to docs v6 assets of X.

Reply:
I may be missing something but Material UI v6 docs <-> MUI X v7 docs, e.g. on current mui.com the associated MUI X version is v7. Shouldn't we keep this parity?

Reply:
@alexfauquette asked me to keep these as they are in the docs' `v6` branch: https://github.com/mui/material-ui-docs/commit/e963c3d0258ae647a645049b30c21accabe3a3a0

I guess the options are:

- v6.mui.com pointing to v6 X docs, which is consistent but breaks Material UI v6 docs <-> MUI X v7 docs
- v6.mui.com pointing to v6 core and v7 X, which might be unexpected for users but keeps Material UI v6 docs <-> MUI X v7 docs

None are great options, but it's what we have given our docs and domain structure.

Reply:
If you don't do that, the `v6.mui.com/x/` will display the docs of the v7 and the v6 will be impossible to reach